Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -288,4 +289,14 @@ id valueAsID() const
     runTestNS({ [NSNumber numberWithInteger: NSIntegerMax] });
     runTestNS({ [NSNumber numberWithInteger: NSIntegerMin] });
     runTestNS({ [NSNumber numberWithUnsignedInteger: NSUIntegerMax] });
+
+    NSError *error1 = [NSError errorWithDomain:@"TestWTFDomain" code:1 userInfo:@{ NSLocalizedDescriptionKey : @"TestWTF Error" }];
+    NSError *error2 = [NSError errorWithDomain:@"TestWTFDomain" code:2 userInfo:@{
+        NSLocalizedDescriptionKey: @"Localized Description",
+        NSLocalizedFailureReasonErrorKey: @"Localized Failure Reason Error",
```

Comment:
It would be good to add an array, a dictionary, a URL, a number, a cert chain and a SecTrustRef with key NSErrorPeerCertificateChainKey, an identity with key NSErrorClientCertificateChainKey, and an unexpected value like a number with key NSErrorClientCertificateChainKey.  We already have functions called testIdentity(), testCertificate(), and secTrustFromCertificateChain that may need some moving to be used here.